docs/Fields.md

```diff
@@ -501,17 +501,18 @@ export const UserList = () => (
 
 **Tip**: In such custom fields, the `source` is optional. React-admin uses it to determine which column to use for sorting when the column header is clicked. In case you use the `source` property for additional purposes, the sorting can be overridden by the `sortBy` property on any `Field` component.
 
-If you build a reusable field accepting a `source` props, you will probably want to support deep field sources (e.g. source values like `author.name`). Use [lodash/get](https://www.npmjs.com/package/lodash.get) to replace the simple object lookup. For instance, for a Text field:
+If you build a reusable field accepting a `source` props, you will probably want to support deep field sources (e.g. source values like `author.name`). Use the [`useFieldValue` hook](/useFieldValue.md) to replace the simple object lookup. For instance, for a Text field:
 
 ```diff
 import * as React from 'react';
-+import get from 'lodash/get';
-import { useRecordContext } from 'react-admin';
+-import { useRecordContext } from 'react-admin';
++import { useFieldValue } from 'react-admin';
 
-const TextField = ({ source }) => {
-    const record = useRecordContext();
+const TextField = (props) => {
+-    const record = useRecordContext();
++   const value = useFieldValue(props);
 -   return record ? <span>{record[source]}</span> : null;
-+   return record ? <span>{get(record, source)}</span> : null;
++   return record ? <span>{value}</span> : null;
 }
 
```

Comment:
Also, you should add that you don't need to check the existence of the `record` before (contrary to what's explained earlier) when using `useFieldValue`.

Reply:
Added